client/src/components/DashBoard/AdminDashboard.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { WhiteSpace } from "antd-mobile";
+import { Input, Select, Table } from "antd";
+import axios from "axios";
+import styled from "styled-components";
+
+import TextAvatar from "components/TextAvatar";
+import { ROLES } from "constants/permissions";
+import { RED, DARK_GRAY } from "constants/colors";
+import { getInitialsFromFullName } from "utils/userInfo";
+
+const { Option } = Select;
+const { Search } = Input;
+
+const AdDashboard = styled(Table)`
+  padding: 1rem;
+  border-radius: 0.2rem;
+  .delete {
+    font-weight: 700;
+    color: ${RED};
```

Comment:
minor: the styling seems not consistent in our project, e.g. do a global search for `color:`

Reply:
yeap minor; should be pulled from theme